app/assets/stylesheets/application.scss

```diff
@@ -101,8 +106,8 @@
 }
 
  .dev-tag{
+  display: inline-block;
```

Comment:
Properties should be ordered background-color, border-color, border-style, border-width, color, display, margin